utils/rpms/daos.spec

```
@@ -78,6 +78,8 @@ BuildRequires: libcurl4
 # have choice for libpsm_infinipath.so.1()(64bit) needed by libfabric1: libpsm2-compat libpsm_infinipath1
 # have choice for libpsm_infinipath.so.1()(64bit) needed by openmpi-libs: libpsm2-compat libpsm_infinipath1
 BuildRequires: libpsm_infinipath1
+# support verbs
+BuildRequires: librdmacm librdmacm-dev
```

```suggestion
BuildRequires: librdmacm-devel
```

The `-devel` package will install the `lib` package.

---

# support verbs
BuildRequires: librdmacm
BuildRequires: librdmacm-devel

Done.